commands/user.go

```diff
@@ -130,6 +130,22 @@ var VerifyUserEmailWithoutTokenCmd = &cobra.Command{
 	Args:    cobra.MinimumNArgs(1),
 }
 
+var UserConvertCmd = &cobra.Command{
+	Use:   "convert (--bot [emails] [usernames] [userIds] | --user <username> --password=PASSWORD [--email=EMAIL])",
```

Comment:
Apologies, I used two different formats in my example for flags but we should pick up one. Maybe `--param <value>` is clearer than `--param=VALUE`, but 0/5 really